deploy/charts/rawfile-csi/templates/storageclass.yaml

```diff
@@ -9,6 +9,8 @@ provisioner: {{ $vals.provisionerName }}
 reclaimPolicy: {{ $class.reclaimPolicy }}
 volumeBindingMode: {{ $class.volumeBindingMode }}
 allowVolumeExpansion: {{ $class.allowVolumeExpansion }}
+parameters:
+  csi.storage.k8s.io/fstype: {{ $class.fsType | default "ext4" }}
```

Comment:
Is it worth validating support fs?

Something like this:
```
{{- $fs := $class.fsType | default "ext4" }}
{{- if (regexMatch "^((ext4|xfs|btrfs))$" $fs) -}}
```

Reply:
I think we should handle that using json schema

Reply:
Yeah, that would be even better :)

Reply:
Will do it in another PR